### PR TITLE
Prevent closing after 10 seconds

### PR DESCRIPTION
**Pull request by davidmhewitt**
Since #25 , the has been seemingly broken as it closes after 10 seconds due to the way the `IS_SERVICE` flag works. Let's just use an indefinite hold as this isn't a dbus activated service, it needs to be listening all the time.

### CODE REVIEW

File: src/Agent.vala [hunk 1]
```diff
@@ -1,6 +1,5 @@
 /*-
- * Copyright (c) 2017 elementary LLC.
- * Copyright (C) 2017 David Hewitt <davidmhewitt@gmail.com>   
+ * Copyright (c) 2017-2019 elementary LLC.
```
Comment (peteruithoven) [inline src/Agent.vala]: From @cassidyjames:
> Also, the legal entity is `elementary, Inc.` now. So if anything is being updated, it should be updated to that.

**Comment by worldofpeace** [comment]
Shouldn't this have fixed https://github.com/elementary/pantheon-agent-geoclue2/issues/26 which was an actual issue?

**Comment by decathorpe** [comment]
@worldofpeace yep, looks like the problem I was seeing on fedora actually was a real issue. surprise surprise :rofl:

**Comment by worldofpeace** [comment]
@decathorpe :rofl: Right, reported issues usually happened because there was some problem.

Good that reports from downstreams can lead to issues like this being noticed and fixed.

**Pull request merged**